fuzzing/cc_deps.bzl

```diff
@@ -21,24 +21,21 @@ load("//fuzzing:common.bzl", "fuzzing_launcher")
 def cc_fuzz_test(
         name,
         srcs,
```

Comment:
Should we allow `src` to be unspecified? Presumably, sometimes the fuzz target source may already be available as a library, so we just need to link that. In that case, we can also drop the `srcs` argument here.